examples/configFiles/deepMedic/train/trainConfig.cfg

```diff
@@ -11,6 +11,8 @@ folderForOutput = "../../../output/"
 #  [Optional] Path to a saved model, to load parameters from at beginning of the session. If one is also specified from command line, the latter will be used.
 #cnnModelFilePath = "../../../output/models/placeholder"
 
+#  [Optional] Log to Tensorboard, set to 0 or remove to suppress logging to tensorboard
```

Comment:
Also, I would either put this right after the "output folder", or at the far end of the file. Not sure.